mappings/net/minecraft/Oxidizable.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5955 net/minecraft/Oxidizable
```

Comment:
Need a package, like it was in `block` before. See https://github.com/FabricMC/yarn/commit/696ae0e310862ea032428dd106448b3f749fa0be#diff-630aea1ba64f09736359a6e96df90b95d6bc66f158378d3bf9197ad2b0ba0e7f

Reply:
Fixed in 8a5e2db4b072358419e72883d7dfd195f64bf50f